client-v2/pom.xml

```diff
@@ -94,13 +102,15 @@
         <dependency>
             <groupId>org.testng</groupId>
             <artifactId>testng</artifactId>
+            <version>${testng.version}</version>
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.projectlombok</groupId>
-            <artifactId>lombok</artifactId>
-            <version>1.18.32</version>
-            <scope>provided</scope>
+            <groupId>${project.parent.groupId}</groupId>
+            <artifactId>clickhouse-client</artifactId>
+            <version>${revision}</version>
+            <type>test-jar</type>
+            <scope>test</scope>
```

Comment:
yes, we have client as dependency 
```
        <dependency>
            <groupId>${project.parent.groupId}</groupId>
            <artifactId>clickhouse-client</artifactId>
            <version>${revision}</version>
        </dependency>
```

There is a problem with getting test code as artifact because tests are not part production. This particular dependency tells mvn to grab test code from the client project.